examples/index.json

```diff
@@ -1467,6 +1467,13 @@
         "verifyBuild": true,
         "verifyOutput": true,
         "isLearnByExample": true
+      },
+      {
+        "name": "Alternate receive",
```

Comment:
Correct order is https://github.com/ballerina-platform/ballerina-dev-website/issues/8856#issuecomment-2044468508